trie/zk_trie.go

```diff
@@ -68,15 +68,23 @@ func NewZkTrie(root common.Hash, db *Database) (*ZkTrie, error) {
 	return &ZkTrie{tr, db}, nil
 }
 
-// Get returns the value for key stored in the trie.
-// The value bytes must not be modified by the caller.
-func (t *ZkTrie) Get(key []byte) []byte {
-	sanityCheckByte32Key(key)
-	res, err := t.TryGet(key)
+func (t *ZkTrie) GetNode(path []byte) ([]byte, int, error) {
+	panic("not implemented")
+}
```

Comment:
Is it for interface? I prefer to clarify which interface is implemented. It's hard to track the interface that the struct implement.

Reply:
The implementation is in the snapsync commit.  For this commit, i'll remove it as it seems unnecessary.